tests/test_dspace_dim.py

```diff
@@ -0,0 +1,294 @@
+        edition=None,
+        file_formats=None,
+        format="electronic resource",
+        funding_information=None,
+        identifiers=[Identifier(value="https://hdl.handle.net/1912/2641", kind="uri")],
+        languages=None,
+        locations=None,
+        notes=None,
+        publication_information=None,
+        related_items=None,
+        rights=None,
+        subjects=None,
+        summary=None,
+    )
+
+
+def test_dspace_dim_record_multiple_titles_raises_error(
+    dspace_dim_record_partial, dspace_dim_record_multiple_titles
+):
+    with raises(ValueError):
+        output_records = dspace_dim_record_partial(
+            input_records=dspace_dim_record_multiple_titles
+        )
+        next(output_records)
+
+
+def test_dspace_dim_record_no_citation_field(
+    dspace_dim_record_partial, dspace_dim_record_no_citation_field
+):
+    output_records = dspace_dim_record_partial(
+        input_records=dspace_dim_record_no_citation_field
+    )
+    assert next(output_records) == TimdexRecord(
+        citation=(
+            "Jamerson, James (2002-11): Time lapse movie of meiosis I in a living "
+            "spermatocyte from the crane fly, Nephrotoma suturalis, viewed with "
+            "polarized light microscopy. Woods Hole Oceanographic Institution. "
+            "https://example.com/oai:darchive.mblwhoilibrary.org:1912/2641"
+        ),
+        source="A Cool Repository",
+        source_link="https://example.com/oai:darchive.mblwhoilibrary.org:1912/2641",
+        timdex_record_id="cool-repo:oai:darchive.mblwhoilibrary.org:1912-2641",
+        title=(
+            "Time lapse movie of meiosis I in a living spermatocyte from the crane fly, "
+            "Nephrotoma suturalis, viewed with polarized light microscopy"
+        ),
+        contributors=[
+            Contributor(
+                value="Jamerson, James",
+                affiliation=None,
+                identifier=None,
+                kind="Creator",
+                mit_affiliated=None,
+            ),
+        ],
+        dates=[
+            Date(
+                kind="accessioned", note=None, range=None, value="2009-01-08T16:24:37Z"
+            ),
+            Date(kind="available", note=None, range=None, value="2009-01-08T16:24:37Z"),
+            Date(kind="issued", note=None, range=None, value="2002-11"),
+        ],
+        format="electronic resource",
+        identifiers=[Identifier(value="https://hdl.handle.net/1912/2641", kind="uri")],
+        publication_information=["Woods Hole Oceanographic Institution"],
+    )
+
+
+def test_dspace_dim_record_no_title_raises_error(
+    dspace_dim_record_partial, dspace_dim_record_no_title
+):
+    with raises(ValueError):
+        output_records = dspace_dim_record_partial(
+            input_records=dspace_dim_record_no_title
+        )
+        next(output_records)
+
+
+def test_dspace_dim_record_unqualified_identifier(
```

Comment:
I'm not sure I understand the purpose of this test?

Reply:
It is unnecessary after the last round of changes